src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/ProjectSystem/DotNetProjectHost.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using MonoDevelop.Projects;
+
+namespace Microsoft.VisualStudio.Mac.LanguageServices.Razor.ProjectSystem
+{
+    internal abstract class DotNetProjectHost
```

Comment:
Could technically get rid of this guy and the factory sine we can't really test the types anyways but figured I'd leave them around for consistency sake (that's what we do normally). If you feel differently I don't have a problem removing them.